src/localisation/string_ids.h

```diff
@@ -1165,6 +1165,8 @@ enum {
 	STR_CONSTRUCTION_RIGHTS_NOT_FOR_SALE = 1727,
 	STR_CANT_BUY_CONSTRUCTION_RIGHTS_HERE = 1728,
 	STR_LAND_NOT_OWNED_BY_PARK = 1729,
+	STR_LAND_NOT_OWNED_AND_NOT_AVAILABLE = 5933,
```

Comment:
I placed it there because they are very related, but I've moved everything down now.